x-pack/plugins/snapshot_restore/public/application/sections/home/snapshot_list/components/snapshot_search_bar.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { SearchFilterConfig } from '@elastic/eui/src/components/search_bar/search_filters';
+import { SchemaType } from '@elastic/eui/src/components/search_bar/search_box';
+import { EuiSearchBarOnChangeArgs } from '@elastic/eui/src/components/search_bar/search_bar';
+import { EuiButton, EuiCallOut, EuiSearchBar, EuiSpacer, Query } from '@elastic/eui';
+import { SnapshotDeleteProvider } from '../../../../components';
+import { SnapshotDetails } from '../../../../../../common/types';
+import { getQueryFromListParams, SnapshotListParams, getListParams } from '../../../../lib';
+
+const moreThanOneSearchClauseError = i18n.translate(
+  'xpack.snapshotRestore.snapshotList.table.moreThanOneSearchClauseError',
+  {
+    defaultMessage: 'More than 1 clause in the search bar',
```

Comment:
I think this error should state the rule that's being violated. For example, "Only one clause in the search bar is allowed". CC @jrodewig

Reply:
I agree. CJ's suggestion is good. We could also use something like:

```suggestion
    defaultMessage: 'You can only use one clause in the search bar',
```